StoreFaceTagSql/run.csx

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using System.Data;
+using System.Data.SqlClient;
+using CM = System.Configuration.ConfigurationManager;
+using Dapper;
+
+public static readonly string connectionString = CM.ConnectionStrings["Atmosphere"].ConnectionString;
+public static async Task Run(string mySbMsg, TraceWriter log)
+{
+    var now = DateTime.UtcNow;
+    var SlackInput = JsonConvert.DeserializeObject<SlackNotification>(mySbMsg);
+    SlackInput.Time = now;
+    log.Info($"Triggered StoreFaceTagSql by: {SlackInput.FaceId}");
```

Comment:
Can you put log message as a first line: `log.Info($"Triggered StoreFaceTagSql by: {mySbMsg}");`